src/test/java/edu/kis/powp/jobs2d/window/command/CommandManagerWindowCommandChangeObserver.java

```diff
@@ -18,6 +18,7 @@ public String toString() {
 	@Override
 	public void update() {
 		commandManagerWindow.updateCurrentCommandField();
+		commandManagerWindow.addToHistory();
```

Comment:
Please introduce your own observer (do not change the existing one) so both functions are decoupled.